src/Documents.jl

```diff
@@ -61,12 +62,13 @@ end
 ## IndexNode.
 
 struct IndexNode
-    pages    :: Vector{String} # Which pages to include in the index? Set by user.
-    modules  :: Vector{Module} # Which modules to include? Set by user.
-    order    :: Vector{Symbol} # What order should docs be listed in? Set by user.
-    build    :: String         # Path to the file where this index will appear.
-    source   :: String         # Path to the file where this index was written.
-    elements :: Vector         # (object, doc, page, mod, cat)-tuple for constructing links.
+    pages       :: Vector{String} # Which pages to include in the index? Set by user.
+    modules     :: Vector{Module} # Which modules to include? Set by user.
+    order       :: Vector{Symbol} # What order should docs be listed in? Set by user.
+    build       :: String         # Path to the file where this index will appear.
+    source      :: String         # Path to the file where this index was written.
+    working_dir :: Union{Symbol,String}         # Path to the directory where code in this index is executed..
```

Comment:
I might be missing something, but why do we need to store it for every `IndexNode` and `ContentsNode` separately? Especially since those fields don't appear to be used?

Reply:
To be honest, I don't fully understand the pipeline, and just added this wherever I saw `build`. I can do a pass where I remove it everywhere it doesn't break things.

Reply:
I removed from Index- and ContentsNodes. Everything still works!